notebooks/advanced/visualize-benjamini-yekutieli-procedure.ipynb

```diff
@@ -708,7 +317,7 @@
     "if HYPOTHESES_INDEPENDENT:\n",
     "    C = [1] * m\n",
     "else:\n",
-    "    C = [sum([1.0 / i for i in range(1, k + 1)]) for k in K]\n",
+    "    C = [sum([1.0 / k for k in K])] * m\n",
```

Comment:
@MaxBenChrist Do you think you will have some time to have a look into this?

Reply:
Yes, will try to free some time the next days.

Reply:
@MaxBenChrist Gentle reminder :-)